### PR TITLE
Add CRI-O registry configuration documentation

### PR DESCRIPTION
**Pull request by saschagrunert**
Hey there :wave:, this is the first draft of a documentation for the registry configuration with CRI-O.

Please take a look.

### CODE REVIEW

**Comment by saschagrunert** [comment]
Is there anything I can do to fix the validation error: https://travis-ci.org/SUSE/doc-caasp/builds/542167591#L511 ?

File: adoc/admin-crio-registries.adoc [hunk 1]
```diff
@@ -0,0 +1,58 @@
+== Configuring container registries for {crio}
+
+Every registry related configuration needs to be done in the TOML file
+`/etc/containers/registries.conf`. This system-wide configuration exists in two
+versions, whereas the latest version supports extended features like the
```
Comment (r0ckarong) [inline adoc/admin-crio-registries.adoc]: If we mention there are "two version" then please describe what both of these are and which one is relevant to the user.

File: adoc/admin-crio-registries.adoc [hunk 2]
```diff
@@ -0,0 +1,58 @@
+== Configuring container registries for {crio}
+
+Every registry related configuration needs to be done in the TOML file
```
Comment (r0ckarong) [inline adoc/admin-crio-registries.adoc]: Are these changes to the TOML file applied automatically on save or do you need to somehow reload this config?

File: adoc/admin-crio-registries.adoc (hunk 1 above)
Comment (saschagrunert) [reply adoc/admin-crio-registries.adoc]: I will stick to the second version, but we have to check which configuration we ship (I take care of that).

File: adoc/admin-crio-registries.adoc (hunk 2 above)
Comment (saschagrunert) [reply adoc/admin-crio-registries.adoc]: Yes, I added a comment about that.

**Comment by saschagrunert** [comment]
@r0ckarong Please take a look again, I gave it another fresh overhaul. :)

**Comment by saschagrunert** [comment]
Thanks :pray:

**Pull request merged**